app/controllers/passengers_controller.rb

```diff
@@ -56,7 +56,7 @@ def destroy
   def passenger_params
     permitted_params = params.require(:passenger)
                              .permit :name, :address, :email, :phone,
-                                     :wheelchair, :mobility_device, :active,
+                                     :wheelchair, :mobility_device_id, :active,
```

Comment:
Unrelated to this PR, but was definitely a bug. Wouldn't have been possible before to actually update a passenger's mobility device.